project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject puppetlabs/lein-ezbake "1.7.6-SNAPSHOT"
+(defproject puppetlabs/lein-ezbake "1.8.0-SNAPSHOT"
```

Comment:
You'd want this as `1.8.1-SNAPSHOT` right? since it points to whatever is next?

Reply:
Nope, the jenkins job will do that. I just need to get it to 1.8.0-SNAPSHOT so it doesn't publish this as 1.7.6